src/daemon/database/actions.rs

```diff
@@ -255,20 +255,22 @@ pub fn db_insert_new_unconfirmed_vault(
     db_path: &PathBuf,
     wallet_id: u32,
     status: &VaultStatus,
+    received_at: u32,
     deposit_outpoint: &OutPoint,
     amount: &Amount,
     derivation_index: ChildNumber,
 ) -> Result<(), DatabaseError> {
     db_exec(db_path, |tx| {
         let derivation_index: u32 = derivation_index.into();
         tx.execute(
-            "INSERT INTO vaults (wallet_id, status, blockheight, deposit_txid, \
-             deposit_vout, amount, derivation_index, updated_at) \
-             VALUES (?1, ?2, ?3, ?4, ?5, ?6, ?7, strftime('%s','now'))",
+            "INSERT INTO vaults (wallet_id, status, blockheight, received_at, \
+             deposit_txid, deposit_vout, amount, derivation_index, updated_at) \
+             VALUES (?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8, strftime('%s','now'))",
```

Comment:
I think this is super weird that
```
received_at == updated_at
```
does not hold for unconfirmed vaults and that it could be confusing once displayed on the UI. What about setting `updated_at` to the value of `received_at` here ?